### PR TITLE
Improve BenchmarkSelectiveStreamReaders

### PR DESCRIPTION
**Pull request by yingsu00**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by sujay-jain — APPROVED** [review]
LGTM

File: presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java [hunk 1]
```diff
@@ -35,8 +35,8 @@
 import com.google.common.base.Strings;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
-import com.google.common.primitives.Longs;
 import io.airlift.units.DataSize;
+import javafx.util.Pair;
```
Comment (ClarenceThreepwood) [inline presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:39]: javafx is typically not used in this repo - and leads to build incompatibilities for some environments.
Maybe change this to "org.apache.commons.lang3.tuple.Pair" ? @yingsu00 @mbasmanova

Comment (mbasmanova) [reply presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:39]: @ClarenceThreepwood Nice catch. Yes, we shouldn't use javafx.util.Pair. org.apache.commons.lang3.tuple.Pair is better.

Comment (mbasmanova) [reply presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:39]: CC: @sujay-jain

Comment (ClarenceThreepwood) [reply presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java:39]: I've added a patch here that fixes this issue. Please take a look

**Pull request merged**